source/fundamentals/connection/socks.txt

```diff
@@ -0,0 +1,119 @@
+.. _node-connect-socks:
+
+===========================
+Enable SOCKS5 Proxy Support
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances by using
+a SOCKS5 proxy. SOCKS5 is a standardized protocol for connecting
+to network services through a proxy server.
+
+.. tip::
+   
+   To learn more about the SOCKS5 protocol, see the Wikipedia entry on
+   :wikipedia:`SOCKS <w/index.php?title=SOCKS&oldid=1160087146>`.
+
+Install the socks Package
+-------------------------
+
+Starting in version 6.0 of the {+driver-short+}, you must *manually* install
```

Comment:
Suggestion:
I think "manually" is unnecessary since there isn't a contrasting option that the user can perform. E.g. "You must automatically install" is not a possible action.